src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -1,8 +1,158 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" >
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
+
     <l:layout permission="${app.ADMINISTER}" title="${%title}" norefresh="true" type="one-column">
```

Comment:
layout tags are giving us a lot of the default Jenkins look by default, we are also ensuring this is for admins only on permissions attribute